tests/mock_tables/mock_multi_asic.py

```diff
@@ -2,6 +2,52 @@
 from unittest import mock
 
 from sonic_py_common import multi_asic
+from utilities_common import multi_asic as multi_asic_util
+
+mock_intf_table = {
+    '': {
+        'eth0': {
+            2: [{'addr': '10.1.1.1', 'netmask': '255.255.255.0', 'broadcast': '10.1.1.1'}] 
+        }
+    },
+    'asic0': {
+        'Loopback0': {
+            17: [{'addr': '62:a5:9d:f4:16:96', 'broadcast': 'ff:ff:ff:ff:ff:ff'}], 
+            2: [{'addr': '40.1.1.1', 'netmask': '255.255.255.255', 'broadcast': '40.1.1.1'}], 
+            10: [{'addr': 'fe80::60a5:9dff:fef4:1696%Loopback0', 'netmask': 'ffff:ffff:ffff:ffff::/64'}]
+            },
+        'PortChannel0001': {
+            17: [{'addr': '82:fd:d1:5b:45:2f', 'broadcast': 'ff:ff:ff:ff:ff:ff'}], 
+            2: [{'addr': '20.1.1.1', 'netmask': '255.255.255.0', 'broadcast': '20.1.1.1'}], 
+            10: [{'addr': 'aa00::1', 'netmask': 'ffff:ffff:ffff:ffff::/64'}, {'addr': 'fe80::80fd:d1ff:fe5b:452f', 'netmask': 'ffff:ffff:ffff:ffff::/64'}]
+            },
+        'Loopback4096': {
+            2: [{'addr': '1.1.1.1', 'netmask': '255.255.255.0', 'broadcast': '1.1.1.1'}]
+        },
+        'veth@eth1': {
+             2: [{'addr': '192.1.1.1', 'netmask': '255.255.255.0', 'broadcast': '192.1.1.1'}]
+        }
+    },
+    'asic1': {
+        'Loopback0': {
+            17: [{'addr': '62:a5:9d:f4:16:96', 'broadcast': 'ff:ff:ff:ff:ff:ff'}], 
+            2: [{'addr': '40.1.1.1', 'netmask': '255.255.255.255', 'broadcast': '40.1.1.1'}], 
+            10: [{'addr': 'fe80::60a5:9dff:fef4:1696%Loopback0', 'netmask': 'ffff:ffff:ffff:ffff::/64'}]
+            },
```

Comment:
Suggest removing 4 spaces from indent to align with other closing braces.

Reply:
Fixed in latest commit